content/docs/command-reference/import-db.md

```diff
@@ -0,0 +1,178 @@
+# import-db
+
+<admon type="warn" title="Experimental">
+
+This is an experimental command, and is subject to breaking changes.
+
+</admon>
+
+Snapshot a table or a SQL query results from a database into CSV/JSON format.
+
+```usage
+usage: dvc import-db [-h] [-q | -v]
+            [--sql SQL | --table TABLE] [--conn CONN]
```

Comment:
Ideally, this should be:
```suggestion
            (--sql SQL | --table TABLE) --conn CONN
```

But auto-linking seems to be broken with this.

Reply:
I think we should change it in the DVC itself also. We don't use this style. I mean all uppercase CONN or TABLE. Pleas check other commands.

If this `(--sql SQL | --table TABLE)` is correct (why?) - we should also fix first on the DVC side - and adjust auto-linker here, I hope it's not that complicated

but let's check if we do this in some other DVC commands

Reply:
They are a mutually-exclusive argument, and one of them is required. I can change the casings of `TABLE`/`SQL` with metavar (uppercased name is just default, and we do those in many places when we are lazy).

Reply:
I have changed this in the docs. Will handle this in DVC separately.

Reply:
>  and we do those in many places when we are lazy

are there examples? I checked a few commands and did't find it.

anyways, I think we need to update it on the DVC side for all the commands

> They are a mutually-exclusive argument, and one of them is required.

good point, then let's change on the DVC side first and we can fix the website later ... DVC is more important I think

Reply:
```
usage: dvc experiments run [-h] [-q | -v] [-f] [-i] [-s] [-p] [-P] [-R] [--downstream] [--force-downstream] [--pull]
                           [--allow-missing] [--dry] [-k] [--ignore-errors] [-n <name>] [-S [<filename>:]<param_name>=<param_value>]
                           [--queue] [--run-all] [-j <number>] [--temp] [-C COPY_PATHS] [-m MESSAGE]
                           [targets ...]
```

See [-m MESSAGE] and [-c COPY_PATHS]. It's just the default from argparse, so it's easy to miss here. There are other commands like `du`/`artifacts get`/`exp save`/`fetch`/`ls-url`/`ls`/`get`/`import`, etc that does this. :)

Reply:
Fixed this in dvc by https://github.com/iterative/dvc/pull/10226.